ydb/docs/en/core/concepts/_includes/ttl.md

```diff
@@ -176,6 +190,45 @@ The example below shows how to use the `modified_at` column with a numeric type
 
 {% endlist %}
 
+### Enabling eviction to Object Storage for an existing table {#enable-tiering-on-existing-tables}
+
+{% note info %}
+
+This functionality is only available for column-oriented tables. For row-oriented tables, this functionality is under development.
+
+{% endnote %}
+
+In the following example, rows of the table `mytable` will be moved to the bucket described in the external data source `/Root/s3_cold_data` one hour after the time recorded in the column `created_at`, and will be deleted after 24 hours:
+
+{% list tabs group=tool %}
+
+- YQL
+
+  ```yql
+  ALTER TABLE `mytable` SET (TTL = Interval("PT1H") TO EXTERNAL DATA SOURCE `/Root/s3_cold_data`, Interval(PT24H) DELETE ON modified_at AS SECONDS);
+  ```
+
+{% if oss == true %}
+
+- C++
```

Comment:
What about other programming languages?

Reply:
Configuration of tiering is not supported in other SDKs, currently